ja/orm/entities.rst

```diff
@@ -223,8 +223,8 @@ CakePHP の ORM を使うためにエンティティークラスを生成する
     {
         protected function _getFullName()
         {
-            return $this->_properties['first_name'] . '  ' .
-                $this->_properties['last_name'];
+            return $this->_fields['first_name'] . '  ' .
```

Comment:
follow it

- fix typo in viritual property in orm/entities: https://github.com/cakephp/docs/pull/6392/files